woocommerce/wc-admin
# Rename admin notes classes and file names to fit conventions

Fixes #5127

This renames the admin notes file names to suit WP conventions - hypens, not underscores. It also follows @jeffstieler's suggestion to [drop the `WC_Admin_` prefix](https://github.com/woocommerce/woocommerce-admin/issues/5127#issuecomment-692148833) on the note classes.

Note that a `DeprecatedNotes.php` file is included, which creates classes with the deprecated names extending the original, renamed classes. The deprecated classes aren't used internally, all deprecated usages should have been replaced with the new class names. The deprecated classes are for any plugins in the wild that might be referencing the current note classes.

### Detailed test instructions:

- Smoke test notifications
- Delete all notifications from the database, reload the WC home page, and observe that notifications are recreated and loaded without errors
- Run the `wc_admin_daily` cron task and make sure no errors appear in the PHP error log

### Changelog Note:

Dev: Rename admin notes classes and file names to fit conventions
